### ISSUE TITLE
Include language selector into navigation

### ISSUE THREAD

**Issue by insectengine** [quarkusio/quarkusio.github.io#1420]
**Describe the bug**
The Quarkus Community are porting versions of the site into localized languages but the current navigation doesn't take into consideration how users can navigate to the various localized sites.

currently, we have japanese (https://ja.quarkus.io/) but a version in chinese is currently in the works.

**Which pages**
All pages.

**Comment by insectengine** [comment]
@ynojima - Since you commented on the PR for the original commit. I wanted to ask you a question. For better usability, I want to change the world "Japanese" to be written in actual Japanese. So native speakers will find it easier. 

A google search says I should use this  日本語

Is this correct?

**Comment by ynojima** [comment]
Yes, "日本語" is an appropriate translation for "Japanese". Thanks!

**Comment by insectengine** [comment]
Thanks!

**Comment by ynojima** [comment]
If I may ask, it would be great if the language selector link is linked to the corresponding page.

For example, when I click the language selector link in "https://quarkus.io/container-first/", it should navigate to "https://ja.quarkus.io/container-first/" instead of "https://ja.quarkus.io/".

**Comment by ynojima** [comment]
Thank you for clarification. I get it!
